File: 13 - Count the Frequency of Elements/solution.py
def count_frequency(arr):
    # Write your code here
    frequency = {}

    for item in arr:
        frequency[item] = frequency.get(item, 0) + 1

    return frequency

# Test Cases
print(count_frequency(["apple", "banana", "apple", "orange", "banana", "banana"]))

File: 13 - Count the Frequency of Elements/solution.js
function countFrequency(arr) {
  // Write your code here
  let frequency = {};

  for (let item of arr) {
    frequency[item] = (frequency[item] || 0) + 1;
  }

  return frequency;
}

// Test Cases
console.log(
  countFrequency(["apple", "banana", "apple", "orange", "banana", "banana"])
);
